<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayTeamWindow</class>
 <widget class="QMainWindow" name="DisplayTeamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Your Team</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1b1f2a, stop:1 #10131a);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>631</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0F111A;
color: #FFFFFF;
font-family: 'Segoe UI', Arial, sans-serif;
font-size: 12px;
border-radius: 15px;
padding: 15px;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_title">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; font-size: 18px; font-weight: bold; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Your Team Details</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_team_name">
       <property name="styleSheet">
        <string notr="true">color: #A0A8B7; font-weight: bold; padding-bottom: 5px; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Team Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="teamNameDisplay">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_home_stadium">
       <property name="styleSheet">
        <string notr="true">color: #A0A8B7; font-weight: bold; padding-bottom: 5px; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Home Stadium:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="stadiumDisplay">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_goals">
       <property name="styleSheet">
        <string notr="true">color: #A0A8B7; font-weight: bold; padding-bottom: 5px; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Goals:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="goalsDisplay">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_budget">
       <property name="styleSheet">
        <string notr="true">color: #A0A8B7; font-weight: bold; padding-bottom: 5px; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Budget:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="budgetDisplay">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
